MTPot.py

```diff
@@ -170,8 +253,23 @@ def main():
     except MissingConfigField:
         honey_logger.info("Syslog reporting disabled, to enable it add its configuration to the configuration file")
     COMMANDS = config.commands
-    server = gevent.server.StreamServer((config.ip, config.port), MyTelnetHandler.streamserver_handle)
-    honey_logger.info("Listening on %d...", config.port)
+
+    try:
+        the_timeout = config.timeout
+    except MissingConfigField:
+        the_timeout = default_timeout
+
+    try:
+        OVERWRITE_COMMANDS = config.overwrite_commands
+    except MissingConfigField:
+        OVERWRITE_COMMANDS = {}
+        OVERWRITE_COMMANDS_LIST = []
+
+    socket.setdefaulttimeout(the_timeout)
+    custom_pool = CustomPool(config.pool)
+    server = gevent.server.StreamServer((config.ip, config.port), MyTelnetHandler.streamserver_handle, spawn=custom_pool)
+
+    honey_logger.info("Listening on port="+str(config.port)+".ip="+str( config.ip)+" with timeout="+str(the_timeout))
```

Comment:
change to:
honey_logger.info("Listening on %s:%d with timeout=%d", config.ip, config.port, the_timeout)